integrations/gitlab/.port/resources/port-app-config.yaml

```diff
@@ -1,6 +1,35 @@
 createMissingRelatedEntities: true
 deleteDependentEntities: true
 resources:
+  - kind: member
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .username
+          title: .name
+          blueprint: '"gitlabGroupMember"'
+          properties:
+            state: .state
+            locked: .locked
+            link: .web_url
+            email: .email
+  - kind: group
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .full_path
+          title: .name
+          blueprint: '"gitlabGroup"'
+          properties:
+            url: .web_url
+            visibility: .visibility
+            description: .description
+          relations:
+            members: '[.__members[].username]'
```

Comment:
I think adding members should be optional for groups, so customers will be able to decide whether they want to have it or no

Reply:
and need to check that creating a new group, will sync the members correctly

Reply:
fair, how about project -> group, same ?